### PR TITLE
Condition extractor

### PR DESCRIPTION
**Pull request by jafeltra**
# Summary

Adds a new helper function to get full entries of bundles.

## New behavior
None

## Code changes
New function

# Testing guidance
This is really to support the PRs in the other two repos. Using this along with those branches should successfully bundle Conditions.

### CODE REVIEW

File: src/helpers/fhirUtils.js [hunk 1]
```diff
@@ -56,11 +56,26 @@ const getBundleResourcesByType = (bundle, type, context = {}, first) => {
   return first ? null : [];
 };
 
+const getBundleEntriesByResourceType = (bundle, type, context = {}, first) => {
```
Comment (Dtphelan1) [inline src/helpers/fhirUtils.js:59]: DP1 : super small, but could we, for the sake of prettiness, make false the default value for first?

Comment (Dtphelan1) [inline src/helpers/fhirUtils.js:59]: DP1 : super small, but could we, for the sake of prettiness, make false the default value for first?

```suggestion
const getBundleEntriesByResourceType = (bundle, type, context = {}, first=false) => {
```

**Comment by Dtphelan1** [comment]
Bar that small nitpick, this looks great! Nice work, and a super useful helper 👍

File: src/helpers/fhirUtils.js (hunk 1 above)
Comment (kjmahalingam) [reply src/helpers/fhirUtils.js:59]: I'm not really reviewing this PR so feel free to ignore me trespassing, but if we do that, can we make that change for `getBundleResourcesByType` as well?

Comment (Dtphelan1) [reply src/helpers/fhirUtils.js:59]: yes please.

Comment (jafeltra) [reply src/helpers/fhirUtils.js:59]: Sure! I think I was going off of the `getBundleResourcesByType` function, but I can update both!

**Pull request merged**